release notes/v0.53.0.md

```diff
@@ -0,0 +1,295 @@
+k6 `v0.53.0` is here 🎉! This release includes:
+
+- Native ECMAScript modules support
+- New experimental OpenTelemetry metrics output
+- Blob support in experimental websockets module
+- Consolidating cloud features and commands under `k6 cloud`
+- Breaking change: remove magic URL resolutions
+
+## Breaking changes
+
+### Require is now specification compliant and always resolve based on the file it is written in [#3534](https://github.com/grafana/k6/issues/3534)
+
+`require` in k6 used to resolve identifiers based on the current "root of execution" (more on that later). In a lot of cases that aligns with the file the `require` is written in or a file in the same folder, which leads to the same result. In a small subset of cases, this isn't the case.
+
+Every other implementation and more or less by the CommonJS specification - require should always be relative to the file it is written in.
+
+This also aligns with how ESM and dynamic `import` also work. In order to align with them `require` now uses the same underlying implementation.
+
+There was a warning message for the last 2 releases trying to tease out cases where that would be problematic.
+
+<details>
+  <summary>"root of execution" explanation</summary>
+
+This is very much an implementation detail that has leaked and likely a not intended one.
+
+Whenever a file is `require`-ed it become the "root of execution" and both `require` and `open` become relative to it. Once the `require` finishes, the previous "root of execution" gets restored. During not init context execution, the main file is the "root of execution".
+
+Example:
+
+Have 3 files:
+main.js
+```javascript
+const s = require("./A/a.js")
+if (s() != 5) {
+	throw "Bad"
+}
+module.exports.default = () =>{} // just for k6 to not error
+```
+
+/A/a.js:
+```javascript
+module.exports = function () {
+  return require("./b.js");
+}
+```
+/A/b.js
+```javascript
+module.exports = 5
+```
+In this example when `require` is called in `/A/a.js` the `main.js` is once again the "root of execution". If you call the function in `/A/a.js` just after defining it though, it will work as expected.
+
+</details>
+
+You might want to use the newly added `import.meta.resolve()` if you want to create a path that is relevant to the currently calling module. This will let you call it outside a helper class and provide the path to it.
```

Comment:
```suggestion
You can use the newly added `import.meta.resolve()` function if you want to create a path that is relevant to the currently calling module. That will let you call it outside of a helper class and provide the path to it. Refer to [docs](https://grafana.com/docs/k6/next/javascript-api/import.meta/resolve/) for more details.
```

That should be the correct URL once we merge https://github.com/grafana/k6-docs/pull/1691.

Reply:
Isn't using `next` a bad idea? Shouldn't it be `latest` ?

Reply:
Yes, thanks for catching @mstoykov! 🙇